sites/cheerpx/src/content/docs/11-guides/File-System-support.md

```diff
@@ -69,6 +69,69 @@ Files in the WebDevice are accessed relative to the current page's URL. For exam
 > [!note] Note
 > It's important to note that this behavior depends on the current page's URL, as it uses a relative path. For more predictable results, it's recommended to use absolute paths when possible.
 
+### Using `application/octet-stream` for WebDevice directories
+
+To handle files from WebDevices properly, set default_type `application/octet-stream`. This ensures that files are treated as binary data. **Only apply this to WebDevice directories**, or it may cause issues with image loading and JavaScript execution.
+
+**Creating the Webdevice:**
+
+```js
+// The path must match the nginx location block (e.g., /webdevice in the nginx configuration)
+const webDevice = await CheerpX.WebDevice.create("/webdevice");
+```
+
+**Nginx configuration example**:
+
+```nginx
+worker_processes  1;
+error_log   nginx_main_error.log info;
+pid nginx_user.pid;
+daemon off;
+
+events {
+    worker_connections  1024;
+}
+
+http {
+    default_type  application/octet-stream;
+    access_log  nginx_access.log;
+    error_log   nginx_error.log info;
+
+    sendfile        on;
+
+    server {
+        listen       8080;
+        server_name  localhost;
+
+        gzip on;
+        gzip_types application/javascript application/wasm text/plain application/octet-stream;
+
+        charset utf-8;
+
+        location / {
+            root .;
+            index  index.html index.htm;
+            add_header 'Cross-Origin-Opener-Policy' 'same-origin' always;
+            add_header 'Cross-Origin-Embedder-Policy' 'require-corp' always;
+        }
+
+        location /webdevice/ {
+            root .;
+            autoindex on;
+            types { }
+            default_type application/octet-stream;
+            add_header 'Cross-Origin-Opener-Policy' 'same-origin' always;
+            add_header 'Cross-Origin-Embedder-Policy' 'require-corp' always;
```

Comment:
Can we add the headers only once outside of the location blocks, or by nesting location blocks? Unsure if that is supported or not, but let's try to avoid duplication if we can